sunpy/cm/tests/test_cm.py

```diff
@@ -0,0 +1,16 @@
+import sunpy.cm
+
+import pytest
+import sunpy.cm as cm
+import matplotlib.pyplot as plt
+
+
+def test_get_cmap():
+    colormap_list = cm.cmlist
+    for cmap in colormap_list.keys():
+        assert colormap_list[cmap] == cm.get_cmap(cmap)
+
+
+def test_show_colormaps():
+    if cm.show_colormaps() is None:
+        assert True
```

Comment:
Can you explain this test to me?

Reply:
Actually, the issue #210 doesn't contain details, so I was waiting for suggestions to test the function. I know this snippet is testing nothing. Could you please suggest what precisely needs to be checked?

Reply:
Well I am unsure how you would test a show colour map without making it a figure we test the hash against.

Reply:
@dpshelio @Cadair

Reply:
You should create this as a figure test.

Reply:
@nabobalis Are you suggesting to to create a test figure using matplotlib and then to image comparision?

Reply:
Yes

Reply:
It might be alot of effort to make it.

Reply:
@nabobalis I'll try my best. Would let you know if any help is needed.